data_handler/celery_app/celery_conf.py

```diff
@@ -47,18 +47,38 @@
         'task': 'run_liquidable_debt_computation_for_zklend',
         'schedule': crontab(minute=f'*/{CRONTAB_TIME}'),
     },
+    f'run_liquidable_debt_computation_for_nostra_alpha_every_{CRONTAB_TIME}_mins': {
+        'task': 'run_liquidable_debt_computation_for_nostra_alpha',
+        'schedule': crontab(minute=f'*/{CRONTAB_TIME}'),
+    },
+    f'run_liquidable_debt_computation_for_nostra_mainnet_every_{CRONTAB_TIME}_mins': {
+        'task': 'run_liquidable_debt_computation_for_nostra_mainnet',
+        'schedule': crontab(minute=f'*/{CRONTAB_TIME}'),
+    },
+    f'run_liquidable_debt_computation_for_hashstack_v0_every_{CRONTAB_TIME}_mins': {
```

Comment:
Please, comment hashtack tasks temporary and add comments with reason:
1. Interest rate part is not yet implemented

Reply:
Done